incskill_site/static/css/resources.css

```
@@ -0,0 +1,70 @@
+#btns{
+  position: relative;
+  height:5%;
+}
+
+#sec-btn {
+  margin:auto;
+  width:30%;
+  border-radius:50px;
+  background:#233854;
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
```

i know buttons don't inherit font-family automatically (which is weird), but you can replace this with
```suggestion
  font-family: inherit;
```
it looks like you use the same font everywhere, and if you ever change it, you'd have to change it everywhere, but this would automatically change buttons if you change the body font family